modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -265,19 +269,47 @@ protected int processStartTag() throws Exception {
 
 		jspWriter.write("</li>");
 
-		int totalPages = (int)Math.ceil(_totalItems / _activeDelta);
+		int totalPages = (int)Math.ceil(
+			(double)_totalItems / (double)_activeDelta);
```

Comment:
You can just cast one and everything will be casted to double but this is Ok and maybe the SF is enforcing it (hope not), let's leave it this way.

Reply:
I was getting different results than in the [React component](https://github.com/liferay/clay/blob/79801c41aa12f40685b2118467c79421ce9cd492/packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx#L170) without these casts, but I can try again.

Reply:
Don't, no need.